tests/core/data/test_convert.py

```diff
@@ -0,0 +1,44 @@
+"""This module tests that the basic conversion and creation methods in qutip
+work properly with networks."""
+import numpy as np
+import pytest
+import qutip
+from qutip_tensornetwork.core.data import Network
+from qutip import data
+from .conftest import random_complex_network, random_numpy_dense
+
+_network = random_complex_network(2)
```

Comment:
Is there a good reason to create on random network here instead of just creating one in the functions that need them? Global mutable state can result in subtle bugs, so it's best avoided where possible.